### ISSUE TITLE
Local time VS UTC time on graph X axis

### ISSUE THREAD

**Issue by jerryatge** [PredixDev/predix-rmd-ref-app#23]
Problematic App:  Reference App
Context: Bentley Compressor 
Problematic point: graph of the 'Asset Parameter Comparison'

Problem:  Time disparity - The time shown in the 'date/time' range selection is in local time, while the time shown in the X axis of the graph is shown as UTC. 

The time in the X axis of the 'PARAMETER COMPARISON' graph  is shown as UTC time on the time series graph (in Assets Parameter Comparison). While the time in the "date/time" range selection is in Local time. 

Please check attached picture time_issue.jpg showing the issue.
![time_issue](https://cloud.githubusercontent.com/assets/20450302/25986316/edc4742e-3721-11e7-8196-e810aa0050a8.jpg)

**Comment by justinlanglais** [comment]
We did a demo for our Signage Tetra Connect project in Las Vegas few weeks ago and peoples were confused regarding the time disparity (why ?). We will have another demo in a week or 2 and it will be nice to be able to have both in local time. Thank you !

**Comment by jerryatge** [comment]
@turnerth ,  the new version has the same issue, please see below the screen shot:
![time_issue](https://cloud.githubusercontent.com/assets/20450302/26040051/4e8a01aa-3954-11e7-878b-604ca558f4f0.jpg)

**Comment by gstroup** [comment]
Yes, this has been an ongoing issue with the px-vis-timeseries components.  I think it's possible, but I haven't had time to track down exactly where to set the time zone configuration.  Please see this issue:  https://github.com/PredixDev/px-vis-timeseries/issues/32

If you find a solution, let us know.  We should fix it on this repo:  https://github.com/predixdev/predix-webapp-starter
